topgun/tasks/garden-deny-network.yml

```diff
@@ -0,0 +1,11 @@
+---
+platform: linux
+
+image_resource:
+  type: registry-image
+  source:
+    repository: gophernet/netcat
```

Comment:
`busybox` has a minimal version of netcat too, covering the flags we need here

```
nc

    nc [OPTIONS] HOST PORT - connect nc [OPTIONS] -l -p PORT [HOST] [PORT] - listen

    Options:

            -e PROG         Run PROG after connect (must be last)
            -l              Listen mode, for inbound connects
            -n              Don't do DNS resolution
            -s ADDR         Local address
            -p PORT         Local port
            -u              UDP mode
            -v              Verbose
            -w SEC          Timeout for connects and final net reads
            -i SEC          Delay interval for lines sent
            -o FILE         Hex dump traffic
            -z              Zero-I/O mode (scanning)
```

(from https://busybox.net/downloads/BusyBox.html)

might be worth then either going with `busybox` (the container image), or `alpine` (as it has busybox)

wdyt?

thx!

Reply:
Good catch, we just did `docker search netcat`.